css/home.css

```
@@ -0,0 +1,215 @@
+.max-container {
+  max-width: var(--size-max-width);
+  margin: auto;
+}
+
+.header-container {
+  height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.btn-login {
+  background-color: var(--primary-100);
+  color: var(--secondary-gray-100);
+  padding: 12px 23px 12px 23px;
```

P3:
width로 버튼의 너비를 정해주셨으니 padding값은 아래처럼 기입해도 좋을 것 같습니다.
아래처럼 작성하시면 나중에 버튼의 너비를 바꿔야할때, width, padding 속성 중 width값만 바꿔주시면  됩니다.

```suggestion
  padding: 12px 0;
  text-align: center;
```